packages/matrix-identity-server/src/db/index.ts

```
@@ -1,276 +1,225 @@
     private readonly logger: TwakeLogger,
     additionnalTables?: Record<T, string>,
     additionnalIndexes?: Partial<Record<T, string[]>>,
-    additionnalInitializeValues?: Partial<
-      Record<Collections, Array<Record<string, string | number>>>
-    >
+    additionnalInitializeValues?: Partial<Record<Collections, Array<Record<string, string | number>>>>,
   ) {
-    this.cleanByExpires = cleanByExpires
-    let Module
+    this.cleanByExpires = cleanByExpires;
+    let Module;
```

Biome (2.4.9)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)